### DISCUSSION TITLE
Why all the Aqara devices are displayed with their Vendor marked as Xiaomi?

### DISCUSSION THREAD

**Discussion by mrskycriper** [Koenkk/zigbee2mqtt#12474] [General]
When i started searching for my Aqara devices in supported devices list i was really confused because i couldn't find them by Vendor. There is no such Vendor as Aqara or Lumi in database right now. It's not a secret that all Aqara branded devices are made by Lumi and not Xiaomi. In fact all Xiaomi and Mijia branded zigbee devices are also made by Lumi in partnership with Xiaomi. 
So why do we have to mark Aqara devices as Xiaomi and not Aqara (or even Lumi)?

**Comment by cddu33** [comment]
Aquara is Xiaomi

**Reply by mrskycriper** [reply]
> Aquara is Xiaomi

No it’s not. That’s the whole point of my question. Aqara is a brand of Lumi United Technology. They are the actual Vendor, not Xiaomi.

**Comment by Burki24** [comment]
> When i started searching for my Aqara devices in supported devices list i was really confused because i couldn't find them by Vendor. There is no such Vendor as Aqara or Lumi in database right now. It's not a secret that all Aqara branded devices are made by Lumi and not Xiaomi. In fact all Xiaomi and Mijia branded zigbee devices are also made by Lumi in partnership with Xiaomi. So why do we have to mark Aqara devices as Xiaomi and not Aqara (or even Lumi)?

**You`re right. Aqara is a company owned by Lumi United Technology Co., Ltd. It's not Xiaomi!**

But we have the same situation for all Tuya-Devices. Tuya isn't the Vendor of Devices. They're  only the company, producing the software inside the devices. So for me, I also have a problem, calling all devices with tuya-fw as tuya-device as the vendor is often different (like Moes and others). Normaly, we have to differentiate we between the Vendors. We have two files in the converter-list: moes and tuya. Sometimes I have to look in tuya.js, sometimes in moes.js. For me, there's not existing consistence.

I'm  often unsure where to integrate a new converter. All inside"tuya.js" or in the correct vendor-file. "Vendor" describes the producer of the hardware, not the FW-supplier. And it's a little bit confusing to find a Moes-device in the frontend under tuya.

Note: Moes is only taken as an example, we have this problem with many vendors. When I search a Device on zigbee2mqtt.io device list, first of all I search the vendor, but if the searched device is not listed there, I have to look in other vendor-lists too and that is confusing.
